src/ppx/graphics_util.cpp

```diff
@@ -31,6 +32,143 @@
 namespace ppx {
 namespace grfx_util {
 
+namespace {
+
+const uint32_t kYuvWidth  = 3152; // 3000;
+const uint32_t kYuvHeight = 3840; // 3000;
+
+// Start planar image helper functions
```

Comment:
These methods aren't used in this PR, but will be used in a subsequent PR for methods to load planar textures.